libs/geography/src/selectors/geography-all.selector.ts

```diff
@@ -1,19 +1,19 @@
 import { DaffCountry } from '../models/public_api';
-import { DaffCountryEntitySelectors, daffCountryEntitySelectors } from './country-entities.selector';
-import { DaffGeographySelectors, daffGeographySelectors } from './geography.selector';
-import { DaffGeographyFeatureSelector, daffGeographyFeatureStateSelector } from './geography-feature.selector';
+import { DaffCountryEntitySelectors, getDaffCountryEntitySelectors } from './country-entities.selector';
+import { DaffGeographySelectors, getDaffGeographySelectors } from './geography.selector';
+import { DaffGeographyFeatureSelector, getDaffGeographyFeatureStateSelector } from './geography-feature.selector';
 
 export interface DaffGeographyAllSelectors<T extends DaffCountry> extends
   DaffCountryEntitySelectors<T>,
   DaffGeographySelectors,
   DaffGeographyFeatureSelector<T> {}
 
-export const daffGeographyAllSelectors = (() => {
+export const getDaffGeographyAllSelectors = (() => {
```

Comment:
I think this should be named `getDaffGeographySelectors`

Reply:
changed